plugins/sinks/compass/README.md

```diff
@@ -13,8 +13,8 @@ sinks:
       compass-User-Email: meteor@odpf.io
       Header-1: value11,value12
     labels:
-      myCustom: $properties.attributes.myCustomField
-      sampleLabel: $properties.labels.sampleLabelField
+      myCustom: $attributes.myCustomField
```

Comment:
do these notations work?

Reply:
There was an issue regarding the same https://github.com/odpf/meteor/issues/474 and I saw the sample config to verify https://github.com/odpf/meteor/blob/main/plugins/sinks/compass/sink.go#L47.